mobilizon_reshare/config/command.py

```diff
@@ -0,0 +1,6 @@
+import dataclasses
+
+
+@dataclasses.dataclass
+class CommandConfig:
```

Comment:
I'm not sure if it'd be best to put this in Dynaconf's settings. Are there any particular reasons not to?